plugins/hls-refactor-plugin/hls-refactor-plugin.cabal

```diff
@@ -102,7 +102,7 @@ test-suite tests
   hs-source-dirs:   test
   main-is:          Main.hs
   other-modules:    Test.AddArgument
-  ghc-options:      -O0 -threaded -rtsopts -with-rtsopts=-N -Wunused-imports
+  ghc-options:      -O0 -threaded -rtsopts -with-rtsopts=-N -Wno-name-shadowing
```

Comment:
-Wunused-imports is already part of -Wall imported via `import: warnings`.
On the other hand these tests rely on shadowing a lot, so disabling this warning.